ecs_deploy/ecs.py

```diff
@@ -242,7 +242,7 @@ def set_commands(self, **commands):
                     old_value=container.get(u'command')
                 )
                 self._diff.append(diff)
-                container[u'command'] = new_command.split(" ")
+                container[u'command'] = [ command for command in new_command.split(" ") if command ]
```

Comment:
Interesting. I forgot to do that in my original PR.

Reply:
Please reformat it to follow PEP8 (regarding whitespace and line-length) and we can merge it! Thanks @normoes !